### ISSUE TITLE
photoswipe.min.js:4  -- Uncaught TypeError: Cannot read property 'firstChild' of undefined(…)

### ISSUE THREAD

**Issue by ZAJDAN** [dimsemenov/PhotoSwipe#1255]
Hello,
right now I am trying use the Photoswipe, but I am facing a problem:
**Uncaught TypeError: Cannot read property 'firstChild' of undefined(…)    ...photoswipe.min.js:4**

Thanks for any hint

**Comment by svdg** [comment]
I have this problem too, help is very appreciated.

**Comment by svdg** [comment]
This Codepen got me going: https://codepen.io/dimsemenov/pen/ZYbPJM. Very nice photo swiper, awesome piece of work!

**Comment by ZAJDAN** [comment]
so You want to say, that this code works on your website?

**Comment by svdg** [comment]
Yes. Still one question: is there an example with options? I want to hide download and translate option text.

**Comment by fabricionaweb** [comment]
Your first instance argument (`pswpElement`) need to be DOM Element, like a `document.querySelector` or jquery `$('selector')[0]`

**Comment by fatih905** [comment]
Add element

    <div class="pswp" tabindex="-1" role="dialog" aria-hidden="true">
        <div class="pswp__bg"></div>
        <div class="pswp__scroll-wrap">
            <div class="pswp__container">
                <div class="pswp__item"></div>
                <div class="pswp__item"></div>
                <div class="pswp__item"></div>
            </div>
            <div class="pswp__ui pswp__ui--hidden">
                <div class="pswp__top-bar">
                    <div class="pswp__counter"></div>
                    <button class="pswp__button pswp__button--close"></button>
                    <button class="pswp__button pswp__button--share"></button>
                    <button class="pswp__button pswp__button--fs"></button>
                    <button class="pswp__button pswp__button--zoom"></button>
                    <div class="pswp__preloader">
                        <div class="pswp__preloader__icn">
                            <div class="pswp__preloader__cut">
                                <div class="pswp__preloader__donut"></div>
                            </div>
                        </div>
                    </div>
                </div>
                <div class="pswp__share-modal pswp__share-modal--hidden pswp__single-tap">
                    <div class="pswp__share-tooltip"></div>
                </div>
                <button class="pswp__button pswp__button--arrow--left"></button>
                <button class="pswp__button pswp__button--arrow--right"></button>
                <div class="pswp__caption">
                    <div class="pswp__caption__center"></div>
                </div>
            </div>
        </div>
    </div>
